code/src/index.css

```diff
@@ -1,13 +1,19 @@
+@import url("https://fonts.googleapis.com/css2?family=Orbitron&display=swap");
+
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  font-family: "Orbitron", sans-serif;
+  color: rgb(255, 255, 255);
+  background-color: black;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+h1 {
+  font-size: 2rem;
+  font-family: "Orbitron", sans-serif;
+  margin: 0;
+}
```

Comment:
Maybe you can make this a styled component instead (however, we realised that you wanted to do it like this because you imported a font)